### DISCUSSION TITLE
The Go compiler version

### DISCUSSION THREAD

**Discussion by thany** [vercel/turborepo#6825] [Help] [answered]
I'm curious to how or where it selects which version of the Go compiler is being used when installing the Turbo package.

The reason I'm asking, is because our security audit has picked out the Go compiler to be potentially unsafe (I don't have specifics atm), and a newer version fixes it. So in order to figure out how to update Go, I think I first need to know where it's at and how it's being determined which version is installed.

I realise the project will soon start using Rust, but it's not entirely clear if this Rust compiler is going to be used to build Turbo only, or whether it's also to be used to build projects based on Turbo. So if waiting for the Rust compiler is the best option, let me know. I guess this would also depend on the timeline of implementing Rust.

In the mean time, if Go can be updated easily, I'd like to do that either way.

**Comment by anthonyshew** [comment, marked as answer]
Rust has landed!

The Rust implementation is the main path as of 1.11, released a few weeks ago. You can [check out the release post here](https://turbo.build/blog/turbo-1-11-0).

We're planning to remove all Go code from the repo within the next couple minor releases of Turborepo. That removal will be within the next few months.

**Reply by thany** [reply]
> or whether it's also to be used to build projects based on Turbo

So the answer to that is yes 😀

Thanks for the followup.
